packages/next-on-pages/templates/_worker.js/utils/cache-interface.ts

```diff
@@ -0,0 +1,178 @@
+export const SUSPENSE_CACHE_URL = 'INTERNAL_SUSPENSE_CACHE_HOSTNAME';
+
+/**
+ * Gets the cache interface to use for the suspense cache.
+ *
+ * @returns Interface for the suspense cache.
+ */
+export async function getSuspenseCacheInterface(): Promise<CacheInterface> {
+	if (process.env.KV_SUSPENSE_CACHE) {
+		return new KvCacheInterface(process.env.KV_SUSPENSE_CACHE);
```

Comment:
I would like if we used a hardcoded name for bindings, e.g. `KV_SUSPENSE_CACHE`, that people can then add to their project. The only question is, what should they be called.

I am open to changing the name used here - I went with this one as Vercel call the API endpoint the Suspense Cache.

Reply:
I would leave it to follow the same naming as nextjs codebase 🤔